config/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_wallet_config"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Developers <mimblewimble@lists.launchpad.net>"]
 description = "Configuration for epic wallet , a simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic-wallet"
```

Comment:
Change this repository to "https://github.com/EpicCash/epic-wallet"

Reply:
Done